.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseMergedDateTimeExtractor.cs

```diff
@@ -327,6 +327,7 @@ private List<ExtractResult> AddMod(List<ExtractResult> ers, string text)
         {
             foreach (var er in ers)
             {
+                TryMergeModifierToken(er, config.AroundRegex, text);
                 var success = TryMergeModifierToken(er, config.BeforeRegex, text);
```

Comment:
This should happen always even if the previous succeeded, right?

Can you please add a code comment saying this and that it then will cascade on the relative modifiers?

Reply:
The comment will make the code intent clearer. Otherwise new people reading the code will think it's incorrect.

Reply:
comment added